pnpm-lock.yaml

```diff
@@ -8388,6 +8388,111 @@ importers:
       tslib: 1.14.1
       typescript: 5.1.6
 
+  packages/framework/collabspaces:
```

Comment:
I'd probably exclude package lock from merging - it would make it harder to do pulls from origin/main.
I'd think it's better to add it to branch right before merging into main.
Maybe I'm wrong here - not sure.